### ISSUE TITLE
Find a replacement notes workflow

### ISSUE THREAD

**Issue by panicsteve** [panicsteve/ubuntu-20-questions#10]
I use Bear on my Mac and it's indispensable to my work.

Things I like about Bear that I'd like to carry over:

- Centralized, synced repository
- Two pane view with sortable list of notes, and note itself in the other pane
- Markdown support
- Searchable across all notes
- Accessible from phone as well as computer
- Ability to link between notes using note titles
- Existing links automatically update when note titles change

A versioning system or even just plaintext files would be adequate for storage, but lacks the ease of searching and linking.  Markdown would note be rendered.

A web-based solution like DokuWiki comes close, but a web app has much higher latency than a native one.  I've also had trouble finding clean themes that work well across both computer and phone.  Search results were a little hard to parse last time I tried DokuWiki.  For security, I'd likely have to put it behind VPN or something which introduces a significant amount of friction when accessing remotely just to quickly add or search for a note.

**Comment by panicsteve** [comment]
Evernote is not going to work for me.

**Comment by panicsteve** [comment]
Simplenote ticks most of the boxes, but I don't believe it supports note linking, and I'd rather self-host if possible

**Comment by JoshMock** [comment]
I went through this exercise recently and landed on [Joplin](https://joplinapp.org/). It's not perfect, but it checked a lot of boxes for me. I'm syncing to Dropbox right now, with hopes of moving it to a [git-annex](https://git-annex.branchable.com/) workflow I'm (very slowly) trying to get working.
